WordPress/gutenberg
# Query Loop block: remove 'add new post' prompt in the sidebar

Fixes:- https://github.com/WordPress/gutenberg/issues/67177

## What?
This PR removes the "Add New Post" link in the Query Loop block's description.

## Why?
The "Add New Post" link in the Query Loop block directs users to the post editor. While its intent is to guide users to create posts for an empty Query Loop block, it disrupts the user's workflow by taking them away from the editor. This leads to potential confusion as users may not know how to return and continue working on the Query Loop block. Removing the link ensures a more seamless experience.

## How?
The link has been removed from the Query Loop block's description. This adjustment ensures that users are not redirected away from their current editing context.

## Testing Instructions
1. Open a post or page.
2. Insert a query loop block.
3. Observe that the "Add New Post" link is no longer present in the block's description.

## Screenshots or screencast 
![image](https://github.com/user-attachments/assets/d198847f-6a07-425c-85f4-a3d368b2471e)
